examples/pub-sub-channel-messages/page.md

```diff
@@ -0,0 +1,118 @@
+# Sending and receiving messages with Pub/Sub
+
+Enable clients to send and receive messages in a channel.
+
+Messaging enables instant, scalable communication through publishing and subscribing to channels. Messages represent dynamic data payloads that clients publish and subscribe to, powering collaborative applications across realtime newsfeeds, auction applications or financial websites.
```

Comment:
```suggestion
Messaging enables instant, scalable communication through publishing and subscribing to channels. Messages represent data payloads that clients publish and subscribe to. They are the basis of realtime communication in all applications.
```

Reply:
I think it would be worth having a paragraph about channels after this. Maybe something along the lines of:

> Channels are used to separate messages into different topics. They are the building block of creating a realtime application using the publish-subscribe pattern. Channels are also the unit of security and scalability. Whilst billions of messages may be delivered by Ably, clients receive only the messages on the channels they subscribe to.